mkdocs/docs/HPC/index.md

```diff
@@ -1,10 +1,30 @@
-# Welcome
+---
+hide: "toc"
+---
+#  Welcome to the HPC-Ugent documentation
 
-Welcome to the {{ hpcinfra }} documentation!
+---
+
+
+![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-The current view of the documentation is intended for people using **{{ OS }}** as operating system.
-To switch to different operating system (Windows, macOS, Linux), click [here](../#force_select_OS).
+You are viewing documentation intended for people using **{{ OS }}**. 
+Use the OS dropdown in the top bar to switch to a different operating system.
+
+**Quick links**
+
+- [Getting Started](getting_started.md)  | [Getting Access](account.md)
+{%- if site == 'Gent' %}
+- [Recording of HPC-UGent intro](https://www.ugent.be/hpc/en/training/introhpcugent-recording)
+- [Linux Tutorial](./only/gent/linux-tutorial/index.md)
+- [Hardware overview](https://www.ugent.be/hpc/en/infrastructure)
+{%- endif %}
+- [FAQ](FAQ.md) | [Troubleshooting](troubleshooting.md) | [Best practices](./best_practices.md)  
+
```

Comment:
To avoid that the block below ends up next to the image, I would include a couple of hard breaks here:

```suggestion
<br/><br/></br>